topopy/MorseSmaleComplex.py

```diff
@@ -557,3 +562,18 @@ def GetNeighbors(self,idx):
             @ Out, a integer list of neighbors indices
         """
         return self.__amsc.Neighbors(int(idx))
+
+    def ValidCheck(self):
```

Comment:
I am going to rename this method to be `is_single_connected_component` since validCheck is not super descriptive and not technically accurate since one could construct a hierarchy that has multiple connected components. This should be easy enough to generate a 2D case with a k-nearest neighbor graph with a low k. I will try to generate a unit test to test this behavior also.